src/test/java/core/basesyntax/strategy/PurchaseOperationImplTest.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax.strategy;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
```

Comment:
You shouldn't change JUnit version in pom.xml file. But I don't what to do. Ask mentors about this.